doc/release-notes/0.9/0.9.md

```diff
@@ -0,0 +1,184 @@
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.9.0 release notes
+
+These release notes support the [Eclipse OpenJ9 0.9.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.9.0/plan).
+
+
+
+## Binaries and supported environments
+
+The following OpenJDK binaries that contain the OpenJ9 VM are now available from the AdoptOpenJDK community:
+
+- [OpenJDK version 10](https://adoptopenjdk.net/?variant=openjdk10-openj9)
+- [OpenJDK version 8 32-bit Windows]().
+- [OpenJDK version 8 x86-64 Linux (non-compressed references VM for large heaps)]().
+
+Builds are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests at AdoptOpenJDK.
+Not all tests are complete for AIX and Windows platforms.
+
+For a full OpenJ9 support statement, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html)
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.9.0. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.9/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1288">#1288</a></td>
+<td valign="top">SharedClassCacheInfo no longer contains a JVMLEVEL constant</td>
+<td valign="top">OpenJDK10 and later: All platforms</td>
+<td valign="top">SharedClassCacheInfo.getCacheJVMLevel() used to return the JVMLEVEL constant that maps to a Java version number, for example <code>JVMLEVEL_JAVA8</code>. This call now returns only the Java version number, for example <code>10</code> for Java 10.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/378">#378</a></td>
+<td valign="top">DDR support is now available</td>
+<td valign="top">Linux, Windows</td>
+<td valign="top">Ability to diagnose problems with the VM, garbage collector, or JIT.</td>
```

Comment:
Suggest pointing to the documentation, preferably in the OpenJ9 user guide, otherwise https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/systemdump_viewer_overview.html

Reply:
Is http://www.eclipse.org/openj9/docs/tool_jdmpview/ in the OpenJ9 docs a suitable topic to point to? Otherwise it'll have to be to the KC topic you suggested.

Reply:
I had a quick chat about that with @pshipton yesterday and he said that topic was OK.